install_config/upgrading/manual_upgrades.adoc

```diff
@@ -899,6 +899,114 @@ For Pacemaker HA multiple master deployments:
 After the service restarts, the certificate update is complete.
 // end::30to31updatingcerts[]
 
+[[manual-upgrading-logging-components]]
+== Upgrading Logging Components
+
+If you have previously link:../../install_config/aggregate_logging.html[deployed
+logging components] and want to upgrade your EFK stack with new images, you must
+take the following steps to safely upgrade with minimal disruption to your log
+data.
```

Comment:
Is "Logging components" the correct way to call this stuff? Is "the EFK" stack the better term? Maybe it encompasses a few more things?